xml/net_sync.xml

```diff
@@ -80,782 +74,364 @@
+   <para>The file <filename>backup.tar.xz</filename> is copied to
+     <filename>/var/backup/</filename>, the absolute path will be
+     <filename>/var/backup/backup.tar.xz</filename>.</para>
+   <para>
+    Keep in mind to insert the <emphasis>trailing slash</emphasis> after the
+    <filename>/var/backup/</filename> directory! If you do not insert the slash,
+    the file <filename>backup.tar.xz</filename> is copied to <filename>/var/backup</filename>
+    (file) <emphasis>not</emphasis> inside the directory <filename>/var/backup/</filename>!
+   </para>
+   <para>
+    To copy a directory is similar to copy a single file. The following
+    example copies the directory <filename>&exampleuser_plain;/</filename> and
+    its content into the directory <filename>/var/backup/</filename>: </para>
+    <screen>&prompt.user;<command>rsync</command> -avz &exampleuser_plain; /var/backup/</screen>
+    <para> Find the copy in the absolute path
+     <filename>/var/backup/&exampleuser_plain;/</filename>. </para>
+  </sect2>
+  <sect2 xml:id="sec.net.sync.rsync.copy.remote.files-and-dirs">
+   <title>Copying Files and Directories Remotely</title>
+   <para> To copy files from or to remote directories requires an IP address or
+    a domain name. A user name can be optional in case your current user name is
+    equal to the remote user name. </para>
+   <para> To copy the file <filename>file.tar.xz</filename> from your local
+    host to the remote host <systemitem
+     class="ipaddress">&exampledomain1ip;.1</systemitem> with same users (being
+    local and remote), use the following command:
+   </para>
+   <screen>&prompt.user;<command>rsync</command> -avz file.tar.xz  &exampleuser_plain;@&exampledomain1ip;.1:</screen>
+    <para> Depending on what you prefer, these commands are also possible and
+    equivalent: </para>
+   <screen>&prompt.user;<command>rsync</command> -avz file.tar.xz &exampledomain1ip;.1:~
+&prompt.user;<command>rsync</command> -avz file.tar.xz &exampledomain1ip;.1:/home/&exampleuser_plain;</screen>
+    <para> In all cases with standard configuration, you will be asked to
+     enter your passphrase of the remote user. This command will copy the
+     <filename>file.tar.xz</filename> to the home directory of user
+     &exampleuser; (usually <filename>/home/&exampleuser_plain;</filename>).
+    </para>
+   <para>
+    To copy a directory remotely is similar to copy a directory locally.
+    The following example copies the directory <filename>&exampleuser_plain;/</filename> and
+    its content into the remote directory <filename>/var/backup/</filename> on
+    the <systemitem class="ipaddress">&exampledomain1ip;.1</systemitem> host:
+   </para>
+   <screen>&prompt.user;<command>rsync</command> -avz &exampleuser_plain; &exampledomain1ip;.1:/var/backup/</screen>
+   <para>
+    Pretending you have write permissions on the host <systemitem class="ipaddress">&exampledomain1ip;.1</systemitem>, you will find the
```

Comment:
s/Pretending/Assuming
Perhaps "assuming" would be a better word than "pretending"?